lifecycle_multiphase_test.go

```diff
@@ -3,21 +3,19 @@
 
 //go:build coraza.rule.multiphase_evaluation
 
+// Multiphase specific tests
```

Comment:
Not sure if still WIP, but it would be good to add a longer description of why we have this separate file, e.g. what's being tested and why it only applies to multiphase

Reply:
I think this is resolved by the text below isn't @anuraaga ?

Reply:
Yes, initially there was just the first line, marking as solved